install.sh

```diff
@@ -3,7 +3,13 @@
 # Exit if something fails
 set -e
 
-if [[ $(basename "$PWD") !=  "JetBrainsRunner"* ]];then
+# Delete the old install if it exists
+INSTALL_DIR=~${HOME}/JetBrainsRunner
```

Comment:
The script should look in the PWD for the project. In my folder structure, the project is not in $HOME.